plugins/sasl/data/modules/Custom Module/MCDU/pages/500-DATA/503-irs-monitor.lua

```diff
@@ -16,12 +16,44 @@
 
 local THIS_PAGE = MCDU_Page:new({id=503})
 
+function THIS_PAGE:calc(mcdu_data, i)
+    self:set_line(mcdu_data, MCDU_LEFT, i, "<IRS" .. i, MCDU_LARGE, ECAM_WHITE)
+    if ADIRS_sys[i].ir_status == IR_STATUS_IN_ALIGN then
+      self:set_line(mcdu_data, MCDU_LEFT, i+1, " ALIGN TTN " .. ADIRS_sys[i]:get_align_ttn(), MCDU_SMALL, ECAM_GREEN)
+    elseif ADIRS_sys[i].ir_status == IR_STATUS_ALIGNED then
+      self:set_line(mcdu_data, MCDU_LEFT, i+1, string.format(" NAV   DRIFT  %.2fNM/H", ADIRS_sys[i].ir_drift), MCDU_SMALL, ECAM_GREEN)
+    end
```

Comment:
Don't forget the other cases ;) 
- `IR_STATUS_OFF`
- `IR_STATUS_ATT_ALIGNED`
- `IR_STATUS_FAULT`